_posts/_2021-08-31-building-opensearch-1.1-distributions.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Building OpenSearch 1.1 Distributions"
+authors: 
+  - dblock
+date: 2021-08-31
+categories: 
+  - technical-post
+twittercard:
+  description: "This post details how we produce reliable and repeatable distributions of OpenSearch."
+---
+The OpenDistro for Elasticsearch (ODFE) was essentially a package containing Elasticsearch OSS and a dozen open-source plugins. Its release process began by picking up a stable version of Elasticsearch OSS, incrementing all the plugin version numbers, and going through a development cycle. Plugins went from `-SNAPSHOT` to `-beta`, then `-rc`, then finally to a release.
```

Comment:
- Open Distro (two words) for Elasticsearch
- The initialism ODFE is was never supposed to be externally facing. Can you just "Open Distro" as the short form?
- use the full term for OSS: e.g. "containing open source Elasticsearch"
- The hyphenation of "open-source" is controversial grammatically. The site has standardized on open (space) search.
- "Its release process" -> "Open DIstro's release process"